packages/commonwealth/client/scripts/views/pages/CommunityManagement/Topics/CreateTopicsSection/CreateTopicSection.tsx

```diff
@@ -0,0 +1,181 @@
+import { pluralizeWithoutNumberPrefix } from 'client/scripts/helpers';
+import { useCommonNavigate } from 'client/scripts/navigation/helpers';
+import app from 'client/scripts/state';
+import {
+  useCreateTopicMutation,
+  useFetchTopicsQuery,
+} from 'client/scripts/state/api/topics';
+import { CWCheckbox } from 'client/scripts/views/components/component_kit/cw_checkbox';
+import { CWText } from 'client/scripts/views/components/component_kit/cw_text';
+import { CWValidationText } from 'client/scripts/views/components/component_kit/cw_validation_text';
+import { CWTextInput } from 'client/scripts/views/components/component_kit/new_designs/CWTextInput';
+import { CWButton } from 'client/scripts/views/components/component_kit/new_designs/cw_button';
+import {
+  ReactQuillEditor,
+  createDeltaFromText,
+  getTextFromDelta,
+} from 'client/scripts/views/components/react_quill_editor';
+import { serializeDelta } from 'client/scripts/views/components/react_quill_editor/utils';
+import type { DeltaStatic } from 'quill';
+import React, { useEffect } from 'react';
+import './CreateTopicSection.scss';
+
+export const CreateTopicSection = () => {
+  const { mutateAsync: createTopic } = useCreateTopicMutation();
+  const navigate = useCommonNavigate();
+  const { data: topics } = useFetchTopicsQuery({
+    communityId: app.activeChainId(),
+  });
+
+  const [errorMsg, setErrorMsg] = React.useState<string | null>(null);
+  const [contentDelta, setContentDelta] = React.useState<DeltaStatic>(
+    createDeltaFromText(''),
+  );
+  const [isSaving, setIsSaving] = React.useState<boolean>(false);
+  const [description, setDescription] = React.useState<string>('');
+  const [featuredInNewPost, setFeaturedInNewPost] =
+    React.useState<boolean>(false);
+  const [featuredInSidebar, setFeaturedInSidebar] =
+    React.useState<boolean>(false);
+  const [name, setName] = React.useState<string>('');
+
+  const editorText = getTextFromDelta(contentDelta);
+
+  useEffect(() => {
+    if (!name || !name.trim()) {
+      setErrorMsg('Name must be specified.');
```

Comment:
![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/bac1ae7e-db12-439d-90ac-56cad2f6b55f)

we should get rid of the error below the submit button. All errors should be under the input. Also, wondering if we should use here CWForm with react-hook-form @mzparacha

Reply:
I also think the same. @Miaplacidus would be great if we could use the `<CWForm/>` component here (there are sample implementations in /components-old page an in some community pages)

Reply:
@masvelio @mzparacha 
Error moved below the name input. 
Created a follow-up ticket [here](https://github.com/hicommonwealth/commonwealth/issues/6397) to address using `CWForm` as well as other enhancements.